### ISSUE TITLE
NTR: HSV present in cutaneous swab (Lab: bacteriology, virology, parasitology, fungal)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8055]
New term request HSV present in cutaneous swab (perlma_100809050543): Culture or PCR, from skin or mucous membranes; herpes simplex virus  UMLS: 0744887

**Comment by pnrobinson** [comment]
@MickeySegal
added this for now
Adding new term: 
- ID: HP:6000541
- label: Positive cutaneous herpes simplex virus nucleic acid test
- parents: [Positive skin infectious agent test (HP:0430117)](https://hpo.jax.org/app/browse/term/HP:0430117)
- pmid: PMID:12834453
- def: Detection of cutaneous herpes simplex virus nucleic acid in the skin by a method such as polymerase chain reaction. The sample is usually obtained by cutaneous swabs from skin lesions.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8055
- synonyms:
